packages/styles/src/tokens/_post-theme.scss

```
@@ -5,4 +5,27 @@
 
 :root {
   @include tokens.from(themes.$post-post);
+
+  // temporarily set here until tokens are implemented
+  --post-border-width: 1px;
+  --post-border-color-base: 204, 204, 204;
+  --post-border-opacity: 1;
+  --post-border-color: rgba(204, 204, 204, 1);
+  --post-border-style: solid;
+  --post-border-color-translucent: rgba($black, 0.175);
+  --post-border-color-primary: 13, 110, 253;
+  --post-border-color-secondary: 108, 117, 125;
+  --post-border-color-success: 25, 135, 84;
+  --post-border-color-danger: 220, 53, 69;
+  --post-border-color-warning: 255, 193, 7;
+  --post-border-color-info: 13, 202, 240;
+  --post-border-color-light: 245, 245, 245;
+  --post-border-color-dark: 33, 37, 41;
+  --post-border-color-white: 245, 245, 245;
+  --post-border-radius-sm: 0.25rem;
+  --post-border-radius: 0.375rem;
+  --post-border-radius-lg: 0.5rem;
+  --post-border-radius-xl: 1rem;
+  --post-border-radius-xl: 2rem;
+  --post-border-radius-pill: 50rem;
```

Here you only have need the variables for the base border utility, i.e. the `.border` class. The other utilities will use values from the utility tokens.
```suggestion
  --post-border-width-base: 1px;
  --post-border-color-base: 204, 204, 204;
  --post-border-opacity-base: 1;
  --post-border-style-base: solid;
```